db/migrate/20230504132212_create_like.rb

```diff
@@ -0,0 +1,10 @@
+class CreateLike < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.references :user, null: false, foreign_key: true
```

Comment:
- [ ] Please here make a reference to the author similar what you did to the post migration
`t.references :author, foreign_key: { to_table: 'users' }`